apps/editors/views.py

```diff
@@ -394,6 +412,10 @@ def exclude_admin_only_addons(queryset):
 def _queue(request, TableObj, tab, qs=None, unlisted=False):
     if qs is None:
         qs = TableObj.Meta.model.objects.all()
+
+    if is_limited_reviewer(request):
+        qs = qs.having('waiting_time_hours >=', REVIEW_LIMITED_DELAY_HOURS)
```

Comment:
i.e. this needs a test so that another developer doesn't accidentally break the feature